docs/unit-testing.md

```diff
@@ -0,0 +1,46 @@
+Jetpack includes several [unit tests](https://github.com/Automattic/jetpack/tree/master/tests) that you can run in your
+ local environment before to submit a new Pull Request.
+
+## Prerequisites
+
+To get started, you can follow the instructions [here](https://phpunit.de/getting-started.html) to install PHPUnit on 
+your machine. Once you've done so, you can get the WordPress testing codebase like so:
+
```

Comment:
Hmm, I see that line below actually. In that case, change or remove this wording.